bolt/include/bolt/Core/BinaryFunction.h

```diff
@@ -2256,6 +2260,11 @@ class BinaryFunction {
   /// Returns the last computed hash value of the function.
   size_t getHash() const { return Hash; }
 
+  /// Returns the function GUID from pseudo-probe description of the function.
+  uint64_t getPseudoProbeGUID() const { return PseudoProbeGUID; }
+
+  void setPseudoProbeGUID(uint64_t GUID) { PseudoProbeGUID = GUID; }
```

Comment:
This is function GUID, pseudo-probe doesn't really have a GUID. It's computed from function name: `Function::getGUID(FName)` Suggest rename to `s/PseudoProbe/Function/`for getter/setter/field.